src/app/portlets/dot-edit-content/shared/iframe-edit-mode.css.ts

```diff
@@ -93,4 +88,127 @@ export const EDIT_PAGE_CSS = `
             transform: rotate(360deg);
         }
     }
+    
+    .dotedit-container__toolbar{
+        position:relative;
+        margin:0 0 -26px 8px;
+        display:table;
```

Comment:
Why table?

Reply:
I needed the element to fit the size of the content, and inline-block was not giving me the results I was expecting, table works just fine.